src/lib/download.ts

```diff
@@ -0,0 +1,281 @@
+import { termData } from '$lib/termData'
```

Comment:
I didn't write this code; I just cleaned it up and moved the logic outside of the component

Reply:
The file is here: https://github.com/icssc/AntAlmanac/blob/custom-buttons/src/lib/termData.ts

You can view the TypeScript alias here in the "paths" property: https://github.com/icssc/AntAlmanac/blob/custom-buttons/tsconfig.json

And the build time resolution is handled by Vite in the config: https://github.com/icssc/AntAlmanac/blob/custom-buttons/vite.config.ts